Sources/StreamChat/APIClient/Endpoints/Payloads/PollsPayloads.swift

```diff
@@ -0,0 +1,251 @@
+//
+// Copyright © 2024 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+struct PollPayload: Decodable {
+    let duration: String
+    let poll: PollResponseData
+
+    init(duration: String, poll: PollResponseData) {
+        self.duration = duration
+        self.poll = poll
+    }
+    
+    enum CodingKeys: String, CodingKey, CaseIterable {
+        case duration
+        case poll
+    }
+}
+
+struct QueryPollsPayload: Decodable {
+    let duration: String
+    let polls: [PollResponseData]
+    var next: String?
+    var prev: String?
+
+    init(duration: String, polls: [PollResponseData], next: String? = nil, prev: String? = nil) {
+        self.duration = duration
+        self.polls = polls
+        self.next = next
+        self.prev = prev
+    }
+    
+    enum CodingKeys: String, CodingKey, CaseIterable {
+        case duration
+        case polls
+        case next
+        case prev
+    }
+}
+
+struct PollResponseData: Decodable {
```

Comment:
As mentioned above, ideally, this would be `PollPayload` to keep consistency with the rest of the codebase. 
```suggestion
struct PollPayload: Decodable {
```